docker/README.md

```diff
@@ -1,86 +1,84 @@
-`docker run --network=host --rm temporalio/tctl:latest --ns test-namespace namespace register -rd 1`
+View metrics at `localhost:8080/dashboard`.
+View Temporal web UI at `localhost:8088`.
+Use Temporal CLI with `docker run --network=host --rm temporalio/tctl:latest`.
 
+For example to register new namespace `test-namespace` with 1 retention day:
+```bash
+docker run --network=host --rm temporalio/tctl:latest --ns test-namespace namespace register -rd 1`
+```
 
 Using a pre-built image
 -----------------------
 With every tagged release of the temporal server, there is also a corresponding
 docker image that's uploaded to docker hub. In addition, the release will also
 contain a **docker.tar.gz** file (docker-compose startup scripts). 
-Go [here](https://github.com/temporalio/temporal/releases/latest) to download a latest **docker.tar.gz** 
+[Download](https://github.com/temporalio/temporal/releases/latest) a latest **docker.tar.gz**.
 
 Execute the following
-commands to start a pre-built image along with all dependencies (cassandra/statsd).
+commands to start a pre-built image along with all dependencies (`cassandra`/`statsd`).
 
-```
-tar -xzvf docker.tar.gz
-cd docker
-docker-compose up
+```bash
+$ tar -xzvf docker.tar.gz
+$ cd docker
+$ docker-compose up
 ```
 
 Building an image for any branch and restarting
 -----------------------------------------
 Replace **YOUR_TAG** and **YOUR_CHECKOUT_BRANCH** in the below command to build:
+```bash
+$ git checkout YOUR_CHECKOUT_BRANCH
+$ docker build . -t temporalio/auto-setup:YOUR_TAG --build-arg TARGET=auto-setup
 ```
-cd $GOPATH/src/github.com/temporalio/temporal
-git checkout YOUR_CHECKOUT_BRANCH
-docker build . -t temporalio/auto-setup:YOUR_TAG --build-arg TARGET=auto-setup
-```
-Replace the tag of **image: temporalio/auto-setup** to **YOUR_TAG** in docker-compose.yml .
-Then stop service and remove all containers using the below commands.
-```
-docker-compose down
-docker-compose up
+Replace the tag of **image: temporalio/auto-setup** to **YOUR_TAG** in `docker-compose.yml`.
+Then restart the service using the below commands:
+```bash
+$ docker-compose down
+$ docker-compose up
 ```
 
-Running temporal service with MySQL
+Running Temporal service with MySQL
 -----------------------------------------
 
-Run temporal with MySQL instead of Cassandra, use following commads:
+Run Temporal with MySQL instead of Cassandra, use following commads:
 
-```
-docker-compose -f docker-compose-mysql.yml up
-docker-compose -f docker-compose-mysql.yml down
+```bash
+$ docker-compose -f docker-compose-mysql.yml up
 ```
 
 Please note that SQL support is still in active developement and it is not production ready yet.
 
-Running temporal service with ElasticSearch
+Running Temporal service with ElasticSearch
 -----------------------------------------
 
-Run temporal with ElasticSearch for visibility instead of Cassandra/MySQL
+Run Temporal with ElasticSearch for visibility instead of Cassandra/MySQL:
```

Comment:
Replace with:

Run Temporal with ElasticSearch for enhanced visibility queries.